data/data-pipeline/data_pipeline/etl/sources/census_acs_median_income/etl.py

```diff
@@ -172,6 +172,8 @@ def _transform_state_median_incomes(self) -> pd.DataFrame:
 
     def extract(self) -> None:
         logger.info("Starting three separate downloads.")
+        # TODO we should update this zip to have the geocorr for tracts, not cbgs, although this works as is
```

Comment:
Yeah, we should fix that -- right now it'll provide innaccurate data because of the drop duplicates line.